src/components/summary/Summary.jsx

```diff
@@ -62,7 +62,7 @@ const MemoizedSummary = React.memo(Summary);
 
 const SummaryContainer = ({ open, setOpen }) => {
   const { total, totalItems, orderStatus } = useBreakdown();
-
+  const lol = 'jonny';
```

Comment:
@jonathanzredfern Think this was left from testing.

Reply:
haha oops